python/lvmdrp/functions/skyMethod.py

```diff
@@ -1590,16 +1589,18 @@ def combine_skies(in_rss: str, out_rss, sky_weights: Tuple[float, float] = None)
 
 def quick_sky_subtraction(in_cframe, out_sframe,
                           skip_subtraction=False, skymethod: str = 'farlines_nearcont'):
-    """ Quick sky refinement using the model in the final CFrame
+    """ main sky subtraction routine using Simple Sky method
 
     Parameters
     ----------
     in_cframe : str
         input CFrame file
     out_sframe : str
         output SFrame file
+    skip_subtraction: Boolean, option
+        flag for skipping sky subtraction, not currently useable
```

Comment:
if this isn't being used we should remove it.

Reply:
Agreed, just waiting to see if anyone needs it still.

Reply:
removed in latest commit, also from main in science_reduction